CHANGELOG.md

```diff
@@ -42,6 +42,9 @@
 
 #### Enhancements
 
+* Add `GroupedReporter` command line reporter to get a summary report in console
```

Comment:
Please mention the command line option here, make sure the line is not longer than 80 characters, finish it with a period and make sure there are two trailing whitespaces at the end of the last line:
```suggestion
* Add `console-grouped` command line reporter to get a summary
  report in the console.  
```